Directory.Packages.props

```
@@ -0,0 +1,94 @@
+    <!-- nuget -->
+    <PackageVersion Include="NuGet.Commands" Version="$(NuGetCommandsVersion)" />
+    <PackageVersion Include="NuGet.Frameworks" Version="$(NuGetFrameworksVersion)" />
+    <PackageVersion Include="NuGet.Packaging" Version="$(NuGetPackagingVersion)" />
+    <PackageVersion Include="NuGet.ProjectModel" Version="$(NuGetProjectModelVersion)" />
+    <PackageVersion Include="NuGet.Versioning" Version="$(NuGetVersioningVersion)" />
+    <!-- roslyn -->
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
+    <!-- runtime -->
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjectionVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(MicrosoftExtensionsDependencyInjectionAbstractionsVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyModel" Version="$(MicrosoftExtensionsDependencyModelVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.Logging.Console" Version="$(MicrosoftExtensionsLoggingConsoleVersion)" />
+    <PackageVersion Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageVersion Include="System.Composition" Version="$(SystemCompositionVersion)" />
+    <PackageVersion Include="System.IO.Packaging" Version="$(SystemIOPackagingVersion)" />
+    <PackageVersion Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
+    <PackageVersion Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebVersion)" />
+    <PackageVersion Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
+    <!-- sdk -->
+    <PackageVersion Include="Microsoft.NET.Sdk.WorkloadManifestReader" Version="$(MicrosoftNetSdkWorkloadManifestReaderVersion)" />
+    <!-- xliff-tasks -->
+    <PackageVersion Include="Microsoft.DotNet.XliffTasks" Version="$(MicrosoftDotNetXliffTasksVersion)" />
+  </ItemGroup>
+
+  <!-- external -->
+  <ItemGroup>
+    <PackageVersion Include="Azure.Core" Version="1.35.0" />
+    <PackageVersion Include="Azure.Storage.Blobs" Version="12.18.0" />
+    <PackageVersion Include="CommandLineParser" Version="2.5.0" />
+    <PackageVersion Include="coverlet.collector" Version="1.0.1" />
+    <PackageVersion Include="FluentAssertions" Version="5.10.3" />
+    <PackageVersion Include="Handlebars.Net" Version="1.11.5" />
+    <PackageVersion Include="JetBrains.Annotations" Version="2018.2.1" />
+    <PackageVersion Include="LZMA-SDK" Version="19.0.0" />
+    <PackageVersion Include="McMaster.Extensions.CommandLineUtils" Version="2.3.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.21.0" />
+    <PackageVersion Include="Microsoft.Data.OData" Version="5.8.4" />
```

From the comment above, maybe we want this actually?
I know I've seen it in the code but didn't pay much attention to it.

Maybe we also possibly want to name it `MicrosoftDataODataVersion` instead and use for both?

```suggestion
    <PackageVersion Include="Microsoft.Data.OData" Version="$(MicrosoftDataServicesClientVersion)" />
```

---

Those packages are already deprecated and new versions are unlikely to be released. I think the current state is good enough. I would rather not introduce a new property or tie this together too much.

It would be better to replace any usage of these de-facto deprecated packages.